containers/unit-tests/setup.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh -ex
+
+personality="$1"
+
+dependencies="\
+    appstream-util \
+    autoconf \
+    automake \
+    build-essential \
+    chromium-browser \
+    clang python3 \
+    curl \
+    dbus \
+    gcc-multilib \
+    gdb \
+    git \
+    glib-networking \
+    glib-networking-dbgsym\
+    gtk-doc-tools \
+    intltool \
+    libc6-dbg \
+    libfontconfig1 \
+    libglib2.0-0-dbgsym \
+    libglib2.0-dev \
+    libgnutls28-dev \
+    libgudev-1.0-dev \
+    libjavascript-minifier-xs-perl \
+    libjson-glib-dev \
+    libjson-perl \
+    libkeyutils-dev \
+    libkrb5-dev \
+    liblvm2-dev \
+    libnm-glib-dev \
+    libpam0g-dev \
+    libpcp-import1-dev \
+    libpcp-pmda3-dev \
+    libpcp3-dev \
+    libpolkit-agent-1-dev \
+    libpolkit-gobject-1-dev \
+    libssh-4-dbgsym \
+    libssh-dev \
+    libsystemd-dev \
+    nodejs \
+    npm \
+    pkg-config \
+    pyflakes3 \
+    python3-pep8 \
+    ssh \
+    valgrind \
+    xmlto \
+    xsltproc \
+"
+echo "root:pass" | chpasswd
+
+tee /entrypoint <<EOF
+#!/bin/sh
```

Comment:
Please make all scripts `set -e`. ... if we need this extra script in the first place, it would be nice to just put it all into start.

Reply:
Right.  Good catch.